js/main.js

```diff
@@ -36,6 +39,29 @@ Site = {
   },
 };
 
+Site.Layout = {
+  init: function() {
+    var _this = this;
+
+    _this.$mainContentColumn = $('#main-content').find('.menu-column-content');
```

Comment:
Yh actually thats probably better, I changed to this for a reason that might have just been a typo